middleware/http/wasm/README.md

```diff
@@ -1,5 +1,23 @@
-# WebAssembly Middleware
+## WebAssembly Middleware
 
-Middleware components in this directory load WebAssembly binaries (wasm) to
-modify requests and responses. Each middleware describes its features and
-constraints such as required function exports.
+This component lets you manipulate an incoming request or serve a response with custom logic compiled using the [htp-wasm](https://http-wasm.io/) Application Binary Interface (ABI). The `handle_request` function receives an incoming request and can manipulate it or serve a response as necessary.
+
+Please see the [documentation](https://github.com/dapr/docs/blob/v1.9/daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md) for general configuration.
+
+### Generating Wasm
+
+To compile your wasm, you must compile source using a http-wasm guest SDK such as [TinyGo](https://github.com/http-wasm/http-wasm-guest-tinygo). You can also make a copy of [hello.go](./example/example.go) and replace the `handler.HandleFn` function with your custom logic.
```

Comment:
Better to use [http-wasm-guest-tinygo](https://github.com/http-wasm/http-wasm-guest-tinygo) or something else since TinyGO usually mean the tiny Go Compiler.

Reply:
ok!